frontend/src/app/[locale]/newsletter/confirmation/page.tsx

```diff
@@ -8,7 +8,7 @@ import PageSEO from "src/components/PageSEO";
 import BetaAlert from "src/components/BetaAlert";
 import { useTranslations } from "next-intl";
 import { Metadata } from "next";
-import { getTranslations } from "next-intl/server";
+import { getTranslations, unstable_setRequestLocale } from "next-intl/server";
```

Comment:
Should we add a code comment here that `unstable_setRequestLocale` is a temporary API that next-intl aims to remove it in the future?

Reply:
Maybe we make a ticket for dealing with this in the future?

Reply:
The name `unstable_` is a common convention and speaks for itself, and this is used in every page, so would opt not to comment in the code. I'll create a ticket to remove it. It could be a while based on https://github.com/amannn/next-intl/issues/663